hub/templates/hub/area.html

```diff
@@ -151,6 +151,18 @@ <h5 class="card-title mb-3">Age range</h5>
                     </div>
                 </div>
             </div>
+            <div class="col d-flex flex-column">
```

Comment:
I note the source dataset only covers England, meaning the `constituency_fuel_poverty` context variable is empty for non-English constituencies, resulting in something like this:

<img width="498" alt="Screenshot 2022-11-09 at 11 57 34" src="https://user-images.githubusercontent.com/739624/200825361-a136d441-daaa-434d-831d-ad3e7126c618.png">

Maybe we should wrap this entire `<div class="col…">` in an `{% if constituency_fuel_poverty %}` check?